otx/cli/utils/hpo.py

```diff
@@ -696,6 +696,7 @@ def _prepare_hyper_parameter(self):
     def _prepare_dataset_adapter(self):
         dataset_adapter = get_dataset_adapter(
             self._task.task_type,
+            self._model_template.hyper_parameters.parameter_overrides["algo_backend"]["train_type"]["default_value"],
```

Comment:
So, Intention of this code is to make hpo always use `INCREMENTAL` train type, which is default value. Am I right?

Reply:
No. It is for `train_type`.
I checked when `train_type` is set to `SELFSUPERVISED` in cli, `default_value` is set to `SELFSUPERVISED` although `default_value` is `INCREMENTAL` in `configuration.yaml`.
We can change `default_value` in cli.